### PR TITLE
Translation wigi light api support

### PR DESCRIPTION
**Pull request by kszh**
Added new wigi light api support for translation with "g" and "x" tags. These tags must be unescaped in xliff/json files, but text inside tags must be escaped.

Also these changes suppose that wigi2xliff_light and innovation/components/translationtool/... used only for Lyceum project. Vladimir, please, correct me if I am mistaken.

TODO: update repo of translation tool on tech server
TODO: update Lyceum repo
TODO: write and apply fixup DB for existing Rhapsode translations
TODO: check curator translation tool and email editor (Lyceum repo)
TODO: refactor innovation/components/translationtool to use translations api

@ViktorKurach @pzotov 
FYI

### CODE REVIEW

**Comment by ViktorKurach** [comment]
@kszh thank you for your work! Please notify me when you are merging the PR here and in innovation

File: lib/text/translationutils.flow [hunk 1]
```diff
@@ -1,11 +1,9 @@
-import string;
 import text/languages;
-import text/translation_fixup;
-import text/translationapi; // ContentUnit
 import fs/filesystem; // fileNameOnly
 import formats/json/json;
-import math/stringmath;
+import text/translationapi; // ContentUnit
 import ui/fontmapping; // getLang
+import formats/xml;
```
Comment (vladimirgorky) [inline lib/text/translationutils.flow]: We minimized imports and excluded xml stuff for simple strings. It is why there was simplified xmlEscape function.
Lightweight application itself don't  need xml for translation - it only loads json with simple strings.
To keep this approach I think we need to restore this here and exclude xml import from this utils file.

Comment (alstrup) [reply lib/text/translationutils.flow]: This small amount is OK to have in flow9.

Comment (vladimirgorky) [reply lib/text/translationutils.flow]: Import xml - OK if no problem with app file size now. But my question was not about this import but about functions like this:

xliffNode2WigiLightString(node : XmlNode, level : int, maybeParent : Maybe<XmlNode>) -> string;

This is utilities for wigi format but flow9 as open source knows nothing about wigi so it seems to me it should be placed in innovation repo like translationwikisupport.flow.

**Comment by kszh** [comment]
merged

**Pull request closed without merging**